toolkit/offchain/src/d_param.rs

```diff
@@ -87,15 +80,19 @@ mod tests {
 
 		let body = tx.body();
 		let inputs = body.inputs();
-		// Payment inputs are script utxo and payment utxo
+		// The both inputs are used to cover transaction
```

Comment:
I've changed inputs to smaller ones, so now both are needed here. One is also used as collateral.

Reply:
```suggestion
		// Both inputs are used to cover transaction
```

Reply:
Updated